### ISSUE TITLE
🐛 [BUG] - Better Token Recognition

### ISSUE THREAD

**Issue by 0xIchigo** [helius-labs/xray#261]
### Description

We do not properly display certain tokens under the Tokens page for an account. For example, in the screenshot provided, we do not display any token name or symbol for TheSolanDAO (`7SZUnH7H9KptyJkUhJ5L4Kee5fFAbqVgCHvt7B6wg4Xc`). This shouldn't be the case, as it is a recognized token on Jupiter and [Solscan](https://solscan.io/token/7SZUnH7H9KptyJkUhJ5L4Kee5fFAbqVgCHvt7B6wg4Xc). 

If a token does not have a picture, we should have a fallback picture. We do this when we can't fetch an asset's picture (see the second screenshot)

### Reproduction URL

https://xray.helius.xyz/account/68VhXhZSQxBqC1fBLXnFb2tMiuh3tmiizpZKLTYTwgNG/tokens?network=mainnet

### Reproduction steps

```bash
1. Go to https://xray.helius.xyz/account/68VhXhZSQxBqC1fBLXnFb2tMiuh3tmiizpZKLTYTwgNG/tokens?network=mainnet
2. See error
```

### Screenshots

![image](https://github.com/helius-labs/xray/assets/96965321/acf632cd-b24b-4757-88ef-0f3e1d7b87bb)
![image](https://github.com/helius-labs/xray/assets/96965321/fcb4cf7e-885b-45ec-9590-d07727577b85)

**Comment by kamalbuilds** [comment]
hey @0xIchigo , I would like to try my hands on this issue thanks

**Comment by 0xIchigo** [comment]
Hey @kamalbuilds, go for it! I just assigned the issue over to you

**Comment by 0xIchigo** [comment]
Hey @kamalbuilds, any update?

**Comment by ATREAY** [comment]
gm @0xIchigo,

I attempted to replicate the issue by setting up the repository locally, but the tokens didn't show up. I believe API keys might be needed. Can we discuss this further on Twitter, Discord, or anywhere you prefer?

**Comment by 0xIchigo** [comment]
gm @ATREAY ! 

To build locally, fill in the [following `.env` template](https://github.com/helius-labs/xray/blob/dev/.env.example) with free Helius and Birdeye API keys. That should be everything you need to get started. If you need any additional help, I recommend opening up a support ticket in the Helius Discord

**Comment by shreyanshrathore** [comment]
hey @0xIchigo, I would like to give it a try.

**Comment by shreyanshrathore** [comment]
Hey @0xIchigo, 
Regarding this issue, you've provided a fallback image to load instead of having no image. However, what should be the fallback for the name field? Since the token name isn't visible, should we include any alt text or not?

**Comment by 0xIchigo** [comment]
gm @shreyanshrathore, go for it! The fallback name should be something like "Unrecognized Token"

**Comment by shreyanshrathore** [comment]
Hey @0xIchigo, I have tried my best and fixed the issue. If you have any suggestions or anything that I can do to merge it into the code, please share it with me.

**Comment by 0xIchigo** [comment]
Fixed by #291 — thank you @shreyanshrathore for the contribution!
